### PR TITLE
Fixes #1454 - added note to not claim more than 2 starter issues

### LINKED ISSUE #1454: Create a note in the documentation for not overcommitting
In [CONTRIBUTING.md](https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md) we need to add a note for reminding people to not over-commit themselves. Aka if they currently work on an PR. They should not take another one, except if the PR is in the process of being reviewed. 

1. It helps people to focus on what they have to do.
2. It unblocks other issues for other contributors. 

Cf [discussions on team meeting March 28, 2017](https://wiki.mozilla.org/Compatibility/Meetings/2017-03-28#Outreachy_PR.27s_.2F_issues_.28ola.29)

(It's yet more content. And I know that @zoepage voiced concerns about the length of the documentation and its structure. There's an issue for this. If @zoepage wants to take it over, feel free #264 )

### PR DESCRIPTION
**Pull request by zoepage**
r? @karlcow

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ The logic for the issue tracker is this -
 
 Anything labeled ["status: good-first-bug"](https://github.com/webcompat/webcompat.com/labels/status:%20good%20first%20bug) is perfect for getting started!
 
+> Note: We do not recommend working on more than two `good-first-bugs`. Take it slow and your time to get into the projects own flow. 
+
```
Comment (karlcow) [inline CONTRIBUTING.md]: Semantically this is not a blockquote (aka not a citation from somewhere else but I guess we already butchered this elsewhere.) Suggestion:

I feel the sentence is a bit complicated for non English natives:

> Take it slow and your time to get into the projects own flow.

Take your time. Small steps will help you have a better understanding of the project.

Comment (miketaylr) [inline CONTRIBUTING.md]: > Semantically this is not a blockquote (aka not a citation from somewhere else but I guess we already butchered this elsewhere.)

I just read this out loud, so now it is!

Comment (karlcow) [inline CONTRIBUTING.md]: ahaha. :) cute

**Comment by zoepage** [comment]
@karlcow 👍

**Pull request merged**